src/util/triplea/MapXMLCreator/MapPropertiesPanel.java

```diff
@@ -0,0 +1,360 @@
+package util.triplea.MapXMLCreator;
+
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+import java.awt.event.ActionEvent;
+import java.awt.event.FocusEvent;
+import java.awt.event.FocusListener;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import java.io.File;
+
+import javax.swing.AbstractAction;
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+
+import util.image.FileOpen;
+
+/**
+ * 
+ * @author Erik von der Osten

```

Comment:
Please remove @Author javadoc comment. The information is in Git, and does not help a reader understand the code (which is the point of comments, not to document who the author was at one point in time)